bindings/python/README.md

```diff
@@ -4,14 +4,16 @@ Python binding to the [iota-sdk library](/README.md).
 
 ## Table of contents
 
-- [Requirements](#requirements)
-- [Getting Started](#getting-started)
+- [IOTA SDK Library - Python binding](#iota-sdk-library---python-binding)
+  - [Table of contents](#table-of-contents)
+  - [Requirements](#requirements)
+  - [Getting Started](#getting-started)
     - [Install the IOTA SDK](#install-the-iota-sdk)
-- [Client](#client-usage)
-- [Wallet](#wallet-usage)
-- [Examples](#examples)
-- [API Reference](#api-reference)
-- [Learn More](#learn-more)
+  - [Client Usage](#client-usage)
```

Comment:
Pretty sure we had Usage in the past and removed it because we didn't like it

Reply:
I think it was @Alexandcoats that removed it

Reply:
My bad. Markdown auto docs or some plugin added it. Removed it now.